connect/src/com/telenor/mobileconnect/operatordiscovery/OperatorDiscoveryConfig.java

```diff
@@ -30,30 +30,38 @@ public static Builder builder() {
         return new Builder();
     }
 
-    public static class Builder {
-        OperatorDiscoveryConfig operatorDiscoveryConfig;
+    public static final class Builder {
```

Comment:
private fields are still there for the 

String operatorDiscoveryEndpoint;
etc.